src/api-routes.ts

```
@@ -4975,19 +4975,33 @@ export function registerApiRoutes(
+  // with no config set and throw, aborting plugin registration. Any
+  // /api/aggregate* route can only be reached after start(), so the config
+  // is available by then. Memoized to a single instance so bulk-aggregation
+  // job state survives the start -> progress -> cancel request sequence.
+  let aggregationService: AggregationService | undefined;
+  const getAggregationService = (): AggregationService => {
+    if (!aggregationService) {
+      aggregationService = new AggregationService(
+        {
+          outputDirectory: state.getDataDirPath(),
+          filenamePrefix:
+            state.currentConfig?.filenamePrefix || 'signalk_data',
```

ESLint

[error] Delete `⏎···········`

(prettier/prettier)